### PR TITLE
Add Rationale

### PR DESCRIPTION
**Pull request by ThunderDeliverer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ pragma solidity ^0.8.16;
 interface IERCx is IERC165 {
     /**
      * @notice Used to check whether the given token is soulbound or not.
+     * @dev If this function returns `true`, the transfer of the token MUST be made impossible
```
Comment (steven2308) [inline README.md:57]:
```suggestion
     * @dev If this function returns `true`, the transfer of the token MUST revert execution
```
WDYT?

File: README.md [hunk 2]
```diff
@@ -65,6 +66,15 @@ interface IERCx is IERC165 {
 
 Designing the proposal, we considered the following questions:
 
+1. **Should we propose another Soulbound NFT proposal even if there are already existing ones?**\
+   This proposal aims to provide the minimum necessary specification, as none of the existing proposals have presented the minimal required interface for the NFTs to be soulbound.
+2. **How does this proposal compare to the other proposals?**\
```
Comment (steven2308) [inline README.md:71]: I feel this 2 questions are too similar and repetitive, I would merge them.

**Review by steven2308 — APPROVED** [review]
Minor suggestions

File: README.md (hunk 1 above)
Comment (ThunderDeliverer) [reply README.md:57]: Perfect! 👌

**Pull request merged**